Stripe.xcworkspace/contents.xcworkspacedata

```diff
@@ -78,7 +78,4 @@
          location = "group:IntegrationTester/IntegrationTester.xcodeproj">
       </FileRef>
    </Group>
-   <FileRef
-      location = "group:HTTPStatusError.swift">
-   </FileRef>
```

Comment:
This was an invalid file reference that wasn't associated with any project -- cleaned it up.